appium/tests/specs/mock/setup/RespectsDisallowAttesterSearchForDomainsOnPreDomainBasic.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { MockApi } from 'api-mocks/mock';
+import {
+    SplashScreen,
+} from '../../../screenobjects/all-screens';
+import {attesterPublicKeySamples} from "../../../../api-mocks/apis/attester/attester-endpoints";
+import SetupKeyScreen from "../../../screenobjects/setup-key.screen";
+import MailFolderScreen from "../../../screenobjects/mail-folder.screen";
+import NewMessageScreen from "../../../screenobjects/new-message.screen";
+
+describe('SETUP: ', () => {
+
+  it('respects disallow_attester_search_for_domains on a per-domain basis', async () => {
+
+    const mockApi = new MockApi();
+    mockApi.fesConfig = { clientConfiguration: {flags: ["NO_PRV_CREATE", "NO_PRV_BACKUP", "NO_ATTESTER_SUBMIT", "PRV_AUTOIMPORT_OR_AUTOGEN", "FORBID_STORING_PASS_PHRASE"], key_manager_url: "https://ekm.flowcrypt.com", disallow_attester_search_for_domains: ["disabled.test"]}};
```

Comment:
It'll be better to split this config on multiple lines, as it's quite difficult to see structure when everything on 1 line:
```js
mockApi.fesConfig = { 
    clientConfiguration: {
        flags: ["NO_PRV_CREATE", "NO_PRV_BACKUP", "NO_ATTESTER_SUBMIT", "PRV_AUTOIMPORT_OR_AUTOGEN", "FORBID_STORING_PASS_PHRASE"],
        key_manager_url: "https://ekm.flowcrypt.com", 
        disallow_attester_search_for_domains: ["disabled.test"]
   }
};
```